to.etc.domui/src/main/java/to/etc/domui/component/wizard/messages.properties

```diff
@@ -0,0 +1,4 @@
+wizardstep.default.backbutton=Previous
+wizardstep.default.nextbutton=Next
+wizardstep.default.cancelbutton=Cancel
+wizardstep.default.endbutton=Done
```

Comment:
The usual name for this step is finish, not done, because finish implies that an action is being taken while done does not.